jobs/build/ocp/Jenkinsfile

```diff
@@ -674,16 +674,25 @@ images:build
               catch ( err ) {
                 record_log = buildlib.parse_record_log( OIT_WORKING )
                 builds = record_log['build']
-                failed_map = [:]
+                failed_build_map = [:]
+                failed_push_list = []
                 for(i = 0; i < builds.size(); i++) {
                   bld = builds[i]
                   distgit = bld['distgit']
                   if ( bld['status'] == '-1' ){
-                    failed_map[distgit] = bld['task_url']
+                    failed_build_map[distgit] = bld['task_url']
                   }
                   else {
                     // build may have succeeded later. If so, remove.
-                    failed_map.remove(distgit)
+                    failed_build_map.remove(distgit)
+                  }
+                  
+                  if ( bld['push_status'] == '-1' ){
```

Comment:
Just noticed on this read through, but these status checks should be `if  ... != 0` instead of ==-1 . We may want useful non-zeros at some point.

Reply:
Fair enough. Fixed.